lib/modules/manager/pixi/extract.ts

```diff
@@ -0,0 +1,56 @@
+import { logger } from '../../../logger';
+import { getSiblingFileName, localPathExists } from '../../../util/fs';
+import { Result } from '../../../util/result';
+import type { PackageFileContent } from '../types';
+import { type PixiConfig, PixiToml, PyprojectToml } from './schema';
+
+function getUserPixiConfig(
+  content: string,
+  packageFile: string,
+): null | PixiConfig {
+  if (packageFile.endsWith('pyproject.toml')) {
+    const { val, err } = Result.parse(content, PyprojectToml).unwrap();
+    if (err) {
+      logger.debug({ packageFile, err }, `error parsing ${packageFile}`);
+      return null;
+    }
+
+    return val.tool?.pixi;
+  }
+
+  if (packageFile.endsWith('pixi.toml')) {
+    const { val, err } = Result.parse(content, PixiToml).unwrap();
+    if (err) {
+      logger.debug({ packageFile, err }, `error parsing ${packageFile}`);
+      return null;
+    }
+
+    return val;
+  }
+
+  /* v8 ignore next 2 */
+  return null;
+}
+
+export async function extractPackageFile(
+  content: string,
+  packageFile: string,
+): Promise<PackageFileContent | null> {
+  logger.trace(`pixi.extractPackageFile(${packageFile})`);
+
+  const config = getUserPixiConfig(content, packageFile);
+  if (!config) {
+    return null;
+  }
+
+  const lockfileName = getSiblingFileName(packageFile, 'pixi.lock');
+  const lockFiles: string[] = [];
+  if (await localPathExists(lockfileName)) {
+    lockFiles.push(lockfileName);
+  }
+
+  return {
+    lockFiles,
+    deps: [],
```

Comment:
renovate won't run update artifacts without dependencies.

Reply:
This is not true.

![image](https://github.com/user-attachments/assets/e017429c-e9e4-4354-993e-5760da050c67)
![image](https://github.com/user-attachments/assets/7b6dea38-23ed-4bc5-89bd-9f2ebd2cc29e)

Reply:
okay 🤔